src/internal/biguint.cpp

```diff
@@ -35,3 +55,23 @@ const std::string &BigUInt::getValue() const
     return m_value;
 }
 
+BigUInt BigUInt::operator*(const BigUInt &rhs)
```

Comment:
can you add some unit tests?

Reply:
Added support for `==` operator and tests, as well as for` *` and `divmod`